README.md

```diff
@@ -168,6 +168,8 @@ Configurations that are supported:
   * For this configuration individual channel parameters can be configured. For example, if you define a `client.channel.params` 
 value with `key1=value1_#{topic},key2=value2_#{key}` in your configuration,
 and publish a message to "topic1" with key "key1", the `client.channel.params`  will be configured with `key1=value1_topic1,key2=value2_key1`
+*`message.name`
+  * For example, if you define a `message.name` value with `message_#{topic}_#{key}` in your configuration, and publish a message to "topic1" with key "key1", the `message.name` will be configured with `message_topic1_key1`
```

Comment:
This line should have a blank line beneath it, to vertically space it from the following H2.

Reply:
https://github.com/ably/kafka-connect-ably/pull/48/commits/147fc97609428df73b79724595674a020bcb14bc